protect-api/src/main/java/com/contoso/api/ApiController.java

```diff
@@ -0,0 +1,21 @@
+package com.contoso.api;
+
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(path = "/api")
+public class ApiController {
+
+    // This annotation will trigger MSAL for Java to validate the JWT provided
+    // and also will require that it contains a scope claim of "Greeting.Read"
+    // An invalid or missing JWT token will result in a 401, a token without
+    // sufficent scope will result in a 403.

```

Comment:
```suggestion
    // sufficent scope will result in a 403.
    // The SCOPE_ prefix is a Spring Boot + Azure AD package implementation detail. There is also a ROLE_ prefix.
```

Might as well just pop it in there - I also thought "wut." when I saw this. :)

Reply:
Good call, added in.